samples/openthread/cli/harness/overlay-cert.conf

```diff
@@ -30,7 +30,6 @@ CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y
 
 # Enable and configure Bluetooth LE
 CONFIG_BT=y
-CONFIG_BT_SMP=y
```

Comment:
Is the change in the certification overlay intentional?

Reply:
Yes, the certification tests do not create the bluetooth connection, only advertise. This option enables support for bluetooth secure connection.